src/dmd/glue.d

```diff
@@ -959,7 +960,7 @@ void FuncDeclaration_toObjFile(FuncDeclaration fd, bool multiobj)
         pi++;
     }
 
-    if (target.isPOSIX && fd.linkage != LINK.d && shidden && sthis)
+    if (target.isPOSIX && fd._linkage != LINK.d && shidden && sthis)
```

Comment:
Probably want to use resolvedLinkage here.

Reply:
@kinke This commentary is un-addressed.

Reply:
It is - https://github.com/dlang/dmd/pull/14077#discussion_r866153182. Either I go all the way and revise all sites without any assumptions (the spec says it's either C or Windows, and I'm very confident this never changes and will all be legacy crap once 32-bit Windows support is finally dropped, the only target where it makes a difference), or we keep it as-is - modulo potential renaming(s) (https://github.com/dlang/dmd/pull/14077#issuecomment-1118771801).

Reply:
Yes, sorry, I saw the above comment after I wrote mine.